apps/orams/components/AwardsForm/AwardsForm.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { Form } from 'react-redux-form'
+
+import Layout from 'shared/Layout'
+
+import BaseForm from 'shared/form/BaseForm'
+import SubmitForm from 'shared/form/SubmitForm'
+import ErrorBox from 'shared/form/ErrorBox'
+import MultiInput from 'shared/form/MultiInput.js'
+import formProps from 'shared/formPropsSelector'
+import { loadProfile } from 'orams/actions/profileActions'
+
+class AwardsForm extends BaseForm {
```

Comment:
suggestion, not saying we do it, but these forms be stateless

Reply:
I agree, but should we spend time doing this?